python/lsst/ts/linearstage/csc.py

```diff
@@ -332,5 +356,8 @@ async def do_stop(self, data):
             Command data.
         """
         self.assert_enabled("stop")
-        await self.component.stop()
+        try:
+            await self.component.stop()
+        except Exception:
+            pass

```

Comment:
Again an exception is ignored. Please at least add an inline comment explaining why and what the expected behavior is.